### ISSUE TITLE
Bug: code block line number overlaping text with Code Styler plugin

### ISSUE THREAD

**Issue by Kol9yN** [RyzenFromFire/obsidian-phoenix#2]
Thanks for your theme, love it so much! ❤️ 😍 👍🏻 

Got a serious bug based on theme
Text in code block is overlapped by line number stretched and floated to the right side

Here preview
![Phonix-Conflict-CodeStyler-LineNumber-Overlap](https://github.com/user-attachments/assets/95116070-e288-4eb5-8792-04555065d378)

But its works well when you editing code section (multi-line)
![Phonix-Conflict-CodeStyler-LineNumber-NoOverlap-Edit](https://github.com/user-attachments/assets/f890fa9e-f32f-4197-8b1d-5b718c71fd58)

And Code Styler works perfect with other themes likes Obsidian gruvbox, Material gruvbox, Default Theme
![Phonix-Conflict-CodeStyler-LineNumber-OtherThemes](https://github.com/user-attachments/assets/2af5f824-bcf8-4a08-b61c-21df5e01e3ad)

Here settings of plugin
![Phonix-Conflict-CodeStyler-LineNumber-Settings](https://github.com/user-attachments/assets/1bbae068-ccc9-4bf0-93a4-d3dbad3301d6)

Can you fix it please, because this problem not in a plugin side, and producing to much asspain when reading code snippets

Thank you for your work and attention, hope you will fix it soon

**Comment by Kol9yN** [comment]
Obsidian version 1.6.7
Plugin version 1.1.7
Theme version 2.2.7
All is latest versions for now

**Comment by RyzenFromFire** [comment]
Hello, thanks for your report. I've deduced that this issue is only present in preview mode, which I don't use aside from printing or indirectly via callouts, so I don't often find bugs like that.

The culprit seems to be a `display: block !important;` in the selector `.markdown-preview-section pre code`. This doesn't seem to have any effect on any other part of the theme, so I've removed it in the latest update. The issue should be fixed now.
